src/Apps/Order/Routes/Payment/index.tsx

```diff
@@ -263,7 +263,7 @@ export class PaymentRoute extends Component<PaymentProps, PaymentState> {
             this.onMutationError(
               errors || creditCardOrError.mutationError,
               creditCardOrError.mutationError &&
-                creditCardOrError.mutationError.message
+                creditCardOrError.mutationError.detail
```

Comment:
This fixes an error I saw during the QA session yesterday-- the helpful message from Stripe actually lives in the `detail` in this case.